apple-news.php

```diff
@@ -79,29 +71,20 @@ function apple_news_uninstall_wp_plugin() {
 require __DIR__ . '/includes/class-apple-news.php';
 require __DIR__ . '/admin/class-admin-apple-news.php';
 
-/**
- * Load plugin textdomain.
- *
- * @since 0.9.0
- */
-function apple_news_load_textdomain() {
-	load_plugin_textdomain( 'apple-news', false, __DIR__ . '/lang' );
-}
-add_action( 'plugins_loaded', 'apple_news_load_textdomain' );
-
```

Comment:
Since WordPress 4.7, this is no longer required for plugins hosted in WordPress.org.

The translations comes automatically from here: https://translate.wordpress.org/projects/wp-plugins/publish-to-apple-news/